babel.config.js

```diff
@@ -250,6 +247,10 @@ module.exports = function (api) {
             },
             "flag-BABEL_8_BREAKING",
           ],
+
+          pluginPackageJsonMacro,
+
+          needsPolyfillsForOldNode && pluginPolyfillsOldNode,
```

Comment:
Moved down to only run these two on `src`, since we don't need to run it on our dependencies (and actually, it breaks `core-js-compat` that we bundle in `@babel/standalone`).